lib/base.js

```diff
@@ -7,12 +7,12 @@ var base = module.exports = Common.extend({
         if (node[0] === '#') {
           this.element = this._make('div',{
             id: node.replace(/^#/,''),
-            width: '100%'
+            "style": 'width: 100%;'
```

Comment:
Hi, with the ```<div>``` element in HTML the width attribute is not actually allowed or valid. This is why I updated it to be ```style="width: 100%;"```. This was one of the main issues that is fixed with the PR. Previously the SVG element set the width of the parent div with its viewBox property as the div would "wrap" it's child content.